zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/NotebookRepoSync.java

```diff
@@ -535,4 +528,15 @@ public Note setNoteRevision(String noteId, String revId, AuthenticationInfo subj
     }
     return revisionNote;
   }
+
+  @Override
+  public Boolean isRevisionSupported() {
+    try {
+      return getRepo(0).isRevisionSupported();
+    } catch (IOException e) {
```

Comment:
is IOException sufficient here? we have many different repo implementations..

Reply:
I used this based on one of the previous method ` List<Revision> revisionHistory` implementation
https://github.com/apache/zeppelin/blob/master/zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/NotebookRepoSync.java#L481